.circleci/config.yml

```diff
@@ -0,0 +1,43 @@
+version: 2
+jobs:
+  build:
+    docker: # executor type
+      - image: buildpack-deps:trusty
+    # The secondary container is an instance of the second listed image which is run in a common network where ports exposed on the primary container are available on localhost.
+    steps:
+      - add_ssh_keys:
+          fingerprints: 'Your fingerprint'
```

Comment:
What is the use of this. Please explain. Is it a required item. Cant we read this from environment variable?

Reply:
This is how circle ci add a ssh key to the VM, I'll update the PR

Reply:
I'm using this product for my class, but the repo get too big and Travis CI can't build it in 50mins, So I decide to use Circle CI

Reply:
I'll create a .md file
I think Circle CI also have a limit in there container So I'm going to try more CI

Reply:
Thanks a lot @TGYZ1909. Good to know its limitations. Will add that to the readme soon.

Reply:
And there is another problem For the same repo Travis CI need more than 50mins to build, while Circle CI only need 7mins